### PR TITLE
dev-db/unixODBC: Bump to 2.3.6

### PR DESCRIPTION
**Pull request by jubalh**
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+	if use odbcmanual ; then
+		# We could simply run "make install-html" if we'd not had 
+		# out-of-source builds here.
+		docinto html
+		dodoc -r doc/.
+		find "${ED%/}/usr/share/doc/${PF}/html" -name "Makefile*" -delete || die
+	fi
+
+	use prefix && dodoc README*
```
Comment (mgorny) [inline dev-db/unixODBC/unixODBC-2.3.6.ebuild]: Whaaat? Why?

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		$(use_with unicode iconv-ucode-enc UTF16LE)
+	)
+	ECONF_SOURCE="${S}" econf "${myeconfargs[@]}"
+}
+
+multilib_src_install_all() {
+	einstalldocs
+
+	if use odbcmanual ; then
+		# We could simply run "make install-html" if we'd not had
```
Comment (mgorny) [inline dev-db/unixODBC/unixODBC-2.3.6.ebuild]: Can't you run it inside `mulitlib_src_install` (with `multilib_is_native_abi` conditional)?

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	if use odbcmanual ; then
+		# We could simply run "make install-html" if we'd not had 
+		# out-of-source builds here.
+		docinto html
+		dodoc -r doc/.
+		find "${ED%/}/usr/share/doc/${PF}/html" -name "Makefile*" -delete || die
+	fi
+
+	use prefix && dodoc README*
+	prune_libtool_files
```
Comment (mgorny) [inline dev-db/unixODBC/unixODBC-2.3.6.ebuild]: This is deprecated. Please replace with `find "${D}" -name '*.la' -delete || die`. Then kill ltprune inherit.

**Comment by jubalh** [comment]
Seems someone else already bumped it: https://github.com/gentoo/gentoo/commit/3b7dd8e1b14d90ce35444b47d53ac514bf902949

Without the changes proposed by @mgorny though.

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild (hunk 1 above)
Comment (Polynomial-C) [reply dev-db/unixODBC/unixODBC-2.3.6.ebuild]: Good question. I rolled my git tree back till unixODBC-2.3.1 and the `use prefix && dodoc README*` has already been in there.

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild (hunk 2 above)
Comment (Polynomial-C) [reply dev-db/unixODBC/unixODBC-2.3.6.ebuild]: Would you mind explaining the benefits your suggestions have to the ebuild?

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild (hunk 3 above)
Comment (Polynomial-C) [reply dev-db/unixODBC/unixODBC-2.3.6.ebuild]: Fixed in 82d4fada68514aa14f742cc7e08d5c94907fa099

File: dev-db/unixODBC/unixODBC-2.3.6.ebuild (hunk 2 above)
Comment (mgorny) [reply dev-db/unixODBC/unixODBC-2.3.6.ebuild]: Just answering the comment. I don't know whether it would be better or worse.

**Pull request closed without merging**